.travis.yml

```
@@ -66,8 +67,8 @@ jobs:
 
     # Then build different configurations and targets in parallel.
     - stage: "Main Compilers"
-      env: COMPILER=g++ LOCALIZE=0
-      name: "GCC 5.3 Make build with curses and no Localization"
+      env: COMPILER=g++ MODS=--mods=Magiclysm LOCALIZE=0
```

The id is lower case
```suggestion
      env: COMPILER=g++ MODS=--mods=magiclysm LOCALIZE=0
```

---

Huh.  I assumed it used the directory name which is title case.  Oh well.